### PR TITLE
README: add a paragraph about adding collections

### PR DESCRIPTION
**Pull request by Andersson007**
Relates to https://github.com/ansible-community/ansible-build-data/pull/103

Adds info how to add new collection to a build

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,3 +40,18 @@ may be referred to by other projects and scripts.
   The version specifies the exact version of the collection that appeared in that
   release of Ansible.  This file will be created by the ``antsibull-build single``
   command.
+
+## Adding a new collection
+
+### Next Ansible major release
+
+To add a collection to the next Ansible major release:
```
Comment (felixfontein) [inline README.md:48]:
```suggestion
To add a collection to the next Ansible major release that has not reached feature freeze:
```

Comment (Andersson007) [reply README.md:48]: important point, thanks for spotting! committed

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@Andersson007 thanks for improving the docs!

**Comment by Andersson007** [comment]
@felixfontein thanks for reviewing and merging!

**Pull request merged**